server/src/main/java/org/opensearch/index/shard/StoreRecovery.java

```diff
@@ -347,6 +350,51 @@ void recoverFromRepository(final IndexShard indexShard, Repository repository, A
         }
     }
 
+    void recoverFromRepositoryAndRemoteStore(
```

Comment:
Repository is used in remote store as well. Maybe `recoverFromSnapshotAndRemoteStore`?